build.gradle

```
@@ -13,19 +14,40 @@ repositories {
 }
 
 dependencies {
+	compile group: 'com.google.code.gson', name: 'gson'
 	implementation 'org.springframework.boot:spring-boot-starter-web'
 	implementation 'org.springframework.boot:spring-boot-starter-data-jdbc'
 	implementation 'net.rakugakibox.spring.boot:logback-access-spring-boot-starter:2.7.1'
 	implementation 'pl.allegro.tech.boot:handlebars-spring-boot-starter:0.3.0'
 	implementation 'org.jgrapht:jgrapht-core:1.0.1'
 	implementation 'io.jsonwebtoken:jjwt:0.9.1'
+	compile "javax.xml.bind:jaxb-api"
+	asciidoctor 'org.springframework.restdocs:spring-restdocs-asciidoctor:2.0.4.RELEASE'
+	testImplementation 'org.springframework.restdocs:spring-restdocs-mockmvc:2.0.4.RELEASE'
```

```suggestion
	asciidoctor 'org.springframework.restdocs:spring-restdocs-asciidoctor'
	testImplementation 'org.springframework.restdocs:spring-restdocs-mockmvc'
```

plugins에서 'io.spring.dependency-management'가 어떠한 역할을 하는지 공부해보세요~

---

좋은 지적 감사합니다😁 
`io.spring.dependency-management`가 버전을 자동으로 관리해주고 있었네요😂 
Springboot를 사용하면서 Springboot의 이점을 제대로 활용하지 못하고 있었다니...😱

Spring dependency manager가 관리해주지 않는 라이브러리들을 제외하고는 명시되어있던 버전을 삭제해주었습니다!